rts/Game/UI/KeyBindings.cpp

```diff
@@ -718,7 +703,11 @@ bool CKeyBindings::UnBindAction(const std::string& command)
 {
 	if (debugEnabled)
 		LOG("[CKeyBindings::%s] command=%s", __func__, command.c_str());
-	return RemoveActionFromKeyMap(command, codeBindings) || RemoveActionFromKeyMap(command, scanBindings);
```

Comment:
I read [this](https://en.cppreference.com/w/cpp/language/eval_order) but it still wasn't clear to me whether all operands are evaluated or not and whether behavior differs between pre-C++11 and post. So I 'fixed' it just in case.

Reply:
The second operand of built-in operator `||` is only evaluated if the first is false, so this will be a potential behaviour change.

Reply:
That's good, we want to change the behavior here to be proper. This was a potential bug introduced in the scancodes effort.